dask_cuda/explicit_comms/dataframe/shuffle.py

```diff
@@ -324,6 +414,15 @@ def shuffle(
     rank_to_inkeys = c.stage_keys(name=name, keys=df.__dask_keys__())
     c.client.cancel(df)
 
+    # Get batchsize
+    max_num_inkeys = max(len(k) for k in rank_to_inkeys.values())
+    batchsize = batchsize or dask.config.get("xcomm-batchsize", 1)
+    if batchsize == -1:
+        batchsize = max_num_inkeys
```

Comment:
My understanding is that the effective default before this PR was `batchsize=-1`. Now it will be `batchsize=1`. Is the idea to make the default as stable as possible? (seems fine to me, just asking for my own understanding)